layouts/default.vue

```diff
@@ -76,10 +76,10 @@
           <div class="three wide column">
             <h4 class="ui inverted header"> Join the Community!</h4>
             <div class="ui inverted link list">
+              <a class="item" href="https://discord.com/invite/vESWWrm"><i class="icon discord"></i> Discord</a>
               <a class="item" href="https://ucrcyber.slack.com"><i class="icon slack"></i> Slack</a>
               <a class="item" href="https://linkedin.com/company/cyberucr"><i class="icon linkedin"></i> LinkedIn</a>
               <a class="item" href="https://linkedin.com/company/cyberucr"><i class="icon instagram"></i> Instagram</a>
```

Comment:
Please fix the links here, I see two for linked in, re-add the instagram link

Reply:
issue has been fixed

Reply:
Ty